### ISSUE TITLE
Package installation fails where system preferred locale encoding is ASCII.

### ISSUE THREAD

**Issue by GrahamDumpleton** [bbangert/routes#27]
Due to the presence of a non ASCII character in the CHANGELOG.rst file, installation of the package can fail under Python 3 if the system default encoding for Python is ASCII.

```
Downloading/unpacking routes
  Running setup.py egg_info for package routes
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/data/tests/component_cornice/.tox-0016/py33-with-extensions/build/routes/setup.py", line 9, in <module>
        CHANGES = open(os.path.join(here, 'CHANGELOG.rst')).read()
      File "/data/tests/component_cornice/.tox-0016/py33-with-extensions/lib/python3.3/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 162: ordinal not in range(128)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/data/tests/component_cornice/.tox-0016/py33-with-extensions/build/routes/setup.py", line 9, in <module>

    CHANGES = open(os.path.join(here, 'CHANGELOG.rst')).read()

  File "/data/tests/component_cornice/.tox-0016/py33-with-extensions/lib/python3.3/encodings/ascii.py", line 26, in decode

    return codecs.ascii_decode(input, self.errors)[0]

UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 162: ordinal not in range(128)
```

This appears to correspond to 'á' in 'Alejandro Sánchez'.

**Comment by bbangert** [comment]
I've seen this in other packages (Kazoo) as well. It's rather annoying, is there some other better fix we can put in the code?

**Comment by haypo** [comment]
This issue has already been fixed in Routes 2.1 by the commit 414ba660bccd3eed60f63cc8a1d117740065126f. UTF-8 is now explicitly used to read README.rst & CHANGELOG.rst.

This issue can be closed.

**Comment by webknjaz** [comment]
@bbangert it looks you forgot to close this one
